test/utils/helpers.go

```diff
@@ -93,6 +93,16 @@ func StartOrWaitDeployment(k8sClient client.Client, name, namespace string, repl
 	return nil
 }
 
+// CheckIfConfigured checks if the deployment is configured.
+func CheckIfConfigured(k8sClient client.Client, name, namespace string) error {
```

Comment:
Also, do we need a timeout here?
```suggestion
func CheckDeploment(k8sClient client.Client, name, namespace string) error {
```

Reply:
Done.. Changed to CheckDeploymentConfigured. We just want to check if nephe deployment exist or not. No timeout is needed